src/frontend/screens/Settings/components/LogSettings/index.tsx

```diff
@@ -71,7 +71,11 @@ export default function LogSettings({ isDefault, appName }: LogSettingsProps) {
 
   const getLogContent = () => {
     window.api
-      .getLogContent({ isDefault, appName, defaultLast })
+      .getLogContent({
+        isDefault,
+        appName: isDefault ? '' : appName,
+        defaultLast
+      })
```

Comment:
The backend changed. There are now two arguments instead of three:
https://github.com/BrettCleary/HeroicGamesLauncher/blob/2d2e15b4dfa6c473a1e67e070e6a9e8822f14dbd/src/backend/logger/logfile.ts#L107-L115

I think the `isDefault` is two much.

Reply:
Ah. makes sense. Just changed it